graylog2-web-interface/docs/ux-patterns.md

```diff
@@ -0,0 +1,14 @@
+### Form & Modal Submit Buttons
+
+It is a challenge to unify the placement, stying and naming of form submit buttons across the application.
+Please consider the following best practices when implementing or changing a form submit or cancel button:
```

Comment:
I think it would be easier to read to make this line a simple headline:

`#### Best practises`

I think in general it would be great to start a section in a TLDR like style, so short clear headlines and guidelines, followed by explaining/meaning/thought/context.

Reply:
I like the idea, but I am not sure how to summarise this section even further. I gave it a try, but the TLDR section contins almost the same information like the actual section.

Reply:
I made the section itself a little bit shorter.